SimMuon/GEMDigitizer/src/GEMSimpleModel.cc

```diff
@@ -34,42 +34,29 @@ GEMDigiModel(config)
 , doNoiseCLS_(config.getParameter<bool> ("doNoiseCLS"))
 , fixedRollRadius_(config.getParameter<bool> ("fixedRollRadius"))
 , simulateElectronBkg_(config.getParameter<bool> ("simulateElectronBkg"))
-, simulateLowNeutralRate_(config.getParameter<bool>("simulateLowNeutralRate"))
 , instLumi_(config.getParameter<double>("instLumi"))
 , rateFact_(config.getParameter<double>("rateFact"))
+, referenceInstLumi_(config.getParameter<double>("referenceInstLumi"))
 {
-//initialise parameters from the fit:
-//params for pol3 model of electron bkg for GE1/1:
-  GE11ElecBkgParam0 = 3402.63;
-  GE11ElecBkgParam1 = -42.9979;
-  GE11ElecBkgParam2 = 0.188475;
-  GE11ElecBkgParam3 = -0.0002822;
-//params for expo model of electron bkg for GE2/1:
-  constElecGE21 = 9.74156e+02;
-  slopeElecGE21 = -1.18398e-02;
-//Neutral Bkg
-//Low Rate model L=10^{34}cm^{-2}s^{-1}
-//const and slope for the expo model of neutral bkg for GE1/1:
-  constNeuGE11 = 807.;
-  slopeNeuGE11 = -0.01443;
-//params for the simple pol5 model of neutral bkg for GE2/1:
-  GE21NeuBkgParam0 = 2954.04;
-  GE21NeuBkgParam1 = -58.7558;
-  GE21NeuBkgParam2 = 0.473481;
-  GE21NeuBkgParam3 = -0.00188292;
-  GE21NeuBkgParam4 = 3.67041e-06;
-  GE21NeuBkgParam5 = -2.80261e-09;
-//High Rate model L=5x10^{34}cm^{-2}s^{-1}
-//params for expo model of neutral bkg for GE1/1:
-  constNeuGE11_highRate = 1.02603e+04;
-  slopeNeuGE11_highRate = -1.62806e-02;
-//params for pol5 model of neutral bkg for GE2/1:
-  GE21ModNeuBkgParam0 = 21583.2;
-  GE21ModNeuBkgParam1 = -476.59;
-  GE21ModNeuBkgParam2 = 4.24037;
-  GE21ModNeuBkgParam3 = -0.0185558;
-  GE21ModNeuBkgParam4 = 3.97809e-05;
-  GE21ModNeuBkgParam5 = -3.34575e-08;
+  //Electron Rate: model L=5x10^{34}cm^{-2}s^{-1}
```

Comment:
It is better to be hard-coded. This was agreed previously to keep them here. The reason is that any little change in the parameters will lead to drastically wrong results and it is better to be not touched by mistake.